README.md

```diff
@@ -1 +1,43 @@
-# Lisandro-Seia-s-portfolio
\ No newline at end of file
+Portfolio
+
+> this is my second microverse project
+
+<img src="./assets/imgs/screenshot.png">
```

Comment:
[Optional] Hi there, you have done a great job and your README file looks very professional. I will recommend you put a screenshot of the mobile version since in this milestone we are working on the mobile display. Looking at your screenshot, it the for the desktop display.